src/tool_getparam.c

```diff
@@ -1670,6 +1671,11 @@ ParameterError getparameter(const char *flag, /* f or -long-flag */
           config->socks5_auth &= ~CURLAUTH_GSSAPI;
         break;
 
+      case 'C':
+        /* --tls-use-session-tickets */
+        config->tls_use_session_ticket = TRUE;
```

Comment:
this should be `= toggle` so that the `--no-` prefix works.

Reply:
done